integration_tests/test_plainbox_provider_plugin.py

```diff
@@ -32,3 +32,7 @@ def test_snap_simple_provider(self):
                 project_dir, 'stage', 'providers', 'simple-plainbox-provider',
                 'plainbox-provider-simple.provider'),
             FileExists())
+
+    def test_snap_provider_with_deps(self):
+        project_dir = 'plainbox-provider-with-deps'
+        self.run_snapcraft('prime', project_dir)
```

Comment:
Please make a comment here to justify that the test doesn't have an assertion.
Something like:
# If the dependency can't be found, the prime step will fail.

What about a test for a failed validation?